packages/manager/modules/billing-components/src/components/commitment/translations/Messages_en_GB.json

```diff
@@ -1,7 +1,7 @@
 {
   "billing_commitment_commit": "Subscribe and get a discount",
   "billing_commitment_commit_again": "Re-subscribe and get a discount",
-  "billing_commitment_none": "You do not have a commitment on your {{ serviceName }} subscription. You can get a discount today if you add a commitment to your &lt;dedicated server>{{ serviceType }} subscription.",
+  "billing_commitment_none": "You do not have a commitment on your {{serviceName}} subscription. You can get a discount today if you add a commitment to your {{serviceName}} {{serviceType}} subscription.",
```

Comment:
Shouldn't it be <serviceType> <serviceName> ?

Reply:
They asked for exact text change. So we did not change anything else.